client/sidebar.jsx

```diff
@@ -28,6 +28,15 @@ class Sidebar extends React.Component {
       .catch(() => null);
   }
 
+  categoryStyleBrand() {
+    const text = `${this.state.product.category} > ${this.state.product.style} > ${this.state.product.brand}`;

```

Comment:
This line seems longer than 80 characters.  Could you use the following pattern to make use of multiple lines?

```
variable = `text`
  + `more text`
```

Reply:
Fixed (:D)